cmd/flux/create_source_helm.go

```diff
@@ -60,7 +60,18 @@ For private Helm repositories, the basic authentication credentials are stored i
     --url=https://stefanprodan.github.io/podinfo \
     --cert-file=./cert.crt \
     --key-file=./key.crt \
-    --ca-file=./ca.crt`,
+    --ca-file=./ca.crt
+
+  # Create a source for an OCI-hosted chart
```

Comment:
```suggestion
  # Create a source for an OCI Helm repository
```

The others one need to be adapted to HTTP(s) Helm repository